### ISSUE TITLE
Implement Aspire.NET

### ISSUE THREAD

**Issue by medhatelmasry** [medhatelmasry/GoodBooks#53]
Wire-up the application with Aspire.NET. This will facilitate setup, debugging, and provide telemetry.

**Comment by engelbertlamercanceran** [comment]
@medhatelmasry Are you guys open for help? I'm interested in contributing to this project.

**Comment by medhatelmasry** [comment]
Yes. We would be happy for you to actively contribute to this initiative to
develop an accounting system that can serve the charity sector.

Regards,
Medhat Elmasry
